adoc-mode.el

```diff
@@ -2676,7 +2747,7 @@ new customization demands."
              (null adoc-unichar-alist))
     (adoc-make-unichar-alist))
 
-  (setq adoc-font-lock-keywords (adoc-get-font-lock-keywords))
+  (setq adoc-mode-font-lock-keywords (adoc-get-font-lock-keywords))
```

Comment:
As you can see we intentionally don't use variables named `adoc-mode-something` most of the time, so we can have shorter identifiers. Please, don't rename this variable.